client/src/components/Privacy.style.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  height: 100vh;
+  align-items: center;
+  flex-flow: column wrap;
+  justify-content: center;
+`;
+
+export const H1 = styled.h1`
+  font-weight: bold;
+  font-size: 3rem;
+  background-color: var(--highlight);
+  text-align: center;
+  margin-bottom: 3%;
+  margin-right: 13%;
+`;
```

Comment:
might be useful to ```justify-content: left``` just so that the title is left aligned like in the Figma! :)